packages/connect-ui/src/components/ErrorFallback.tsx

```diff
@@ -1,7 +1,9 @@
 import { IconX } from '@tabler/icons-react';
+import { useMount } from 'react-use';
 
 import { APIError } from '@/lib/api';
 import { triggerClose } from '@/lib/events';
+import { telemetry } from '@/lib/telemetry';
```

Comment:
not sure telemetry is a good name. Since it is limited to send beacon event, why not just `beacon`